src/components/MultiHopTrade/components/TradeInput/components/ConfirmSummary.tsx

```diff
@@ -238,6 +238,10 @@ export const ConfirmSummary = ({
     translate,
   ])
 
+  const buttonText = useMemo(() => {
+    return <Text translation={quoteStatusTranslation} />
+  }, [quoteStatusTranslation])
```

Comment:
unsure what we gain by wrapping this jsx in useMemo, but I aint gonna block you on this